_inc/client/writing/speed-up-site.jsx

```diff
@@ -47,39 +47,54 @@ const SpeedUpSite = withModuleSettingsFormHelpers(
 
 			// If one of them is on, we turn everything off, including Tiled Galleries that depend on Photon.
 			if ( true === siteAcceleratorStatus ) {
+				const messages = {
+					progress: __( 'Disabling Site accelerator…' ),
+					success: __( 'Site accelerator is no longer speeding up your site!' ),
+					error: error => __( 'Error disabling Site accelerator. %(error)s', { args: { error: error } } )
+				};
+				let settings = {};
+
 				if ( false === ! newPhotonStatus && 'active' !== photonStatus ) {
 					newPhotonStatus = false;
-
-					this.props.updateOptions( {
```

Comment:
Comment for the reveiwer: There were two calls to `this.props.updateOptions()` being done here. Now, let's collect the settings, and only then call the action creator once.